geode-pulse/src/main/java/org/apache/geode/tools/pulse/internal/data/Repository.java

```diff
@@ -136,6 +136,9 @@ public Cluster getCluster(String username, String password) {
     }
   }
 
+  public void logoutUser(String username) {
+    clusterMap.remove(username);
```

Comment:
I think we can add the jmConnect.close() method call in here as well:
public void logoutUser(String username){
 Cluster cluster = clusterMap.remove(username);
if(cluster!=null){
  cluster.getJMXConnector().close();
}
}